### DISCUSSION TITLE
Workflows failing

### DISCUSSION THREAD

**Discussion by Xylzlol** [upptime/upptime#801] [Q&A] [answered]
hello! :)

i'm having trouble with my upptime website, https://status.xylz.tv/. The workflows randomly started failing with this error:

Error: The module '/home/runner/work/_actions/upptime/uptime-monitor/v1.29.0/dist/lib/binding/node_libcurl.node'
14was compiled against a different Node.js version using
15NODE_MODULE_VERSION 72. This version of Node.js requires
16NODE_MODULE_VERSION 93. Please try re-compiling or re-installing
17the module (for instance, using `npm rebuild` or `npm install`).

i have no clue how to fix this. any help would be amazing ! :)

**Comment by AnandChowdhary** [comment, marked as answer]
Closing as duplicate of #800
